op.h

```diff
@@ -15,7 +15,12 @@
  *			but this is replaced when op is grafted in, when
  *			this op will point to the real next op, and the new
  *			parent takes over role of remembering starting op.)
+ * 	op_sibparent    Pointer to the op's next sibling, or to the parent
+ *			if there are no more siblings (PERL_OP_PARENT builds)
  *	op_ppaddr	Pointer to current ppcode's function.
+ * 	op_targ         Points to the pad position into which the result of
+ *			the op will be stored. (Unless OPf_STACKED is on, in
+ *			which case the result go onto the value stack.)
```

Comment:
OPf_STACKED usually controls whether an argument arrives on the stack, not whether the result is stacked.

This is otherwise confusing, pp functions typically* won't replace the SV at that entry in the pad, they will typically return that SV, often after using it as a scratch value to avoid creating a new SV each time that particular op is called.

* `pp_refassign` will replace the targ entry, I don't recall any others off hand

Reply:
Hmm, don't know why I got the wrong idea about OPf_STACKED. I had meant that the SV in the pad would be (re)used as a scratch, rather than obliterated, but can see that wording isn't clear enough.

How about this?
"Points to a pad entry (the target) for the current op. The entry typically points to a SV which is (re)used to store the result of the op."

Reply:
I think that's better.